configs/src/networks/networks.ts

```diff
@@ -52,51 +51,40 @@ export const networks: NetworkConfigs = {
     aliases: {
       ethers5: 'maticmum',
       ethers6: 'matic-mumbai',
-      alchemySubgraphs: 'mumbai',
     },
   },
   [SupportedNetworks.BASE]: {
     isTestnet: false,
     chainId: 8453,
     gasPrice: 1000,
     name: SupportedNetworks.BASE,
-    aliases: {
-      alchemySubgraphs: 'base',
-    },
+    aliases: {},
   },
   [SupportedNetworks.BASE_GOERLI]: {
     isTestnet: true,
     chainId: 84531,
     gasPrice: 1000000,
     name: SupportedNetworks.BASE_GOERLI,
-    aliases: {
-      alchemySubgraphs: 'base-testnet',
-    },
+    aliases: {},
   },
   [SupportedNetworks.BASE_SEPOLIA]: {
     isTestnet: true,
     chainId: 84532,
     gasPrice: 1000000,
     name: SupportedNetworks.BASE_SEPOLIA,
-    aliases: {
-      alchemySubgraphs: 'base-sepolia',
-    },
+    aliases: {},
   },
   [SupportedNetworks.ARBITRUM]: {
     isTestnet: false,
     chainId: 42161,
     name: SupportedNetworks.ARBITRUM,
-    aliases: {
-      alchemySubgraphs: 'arbitrum-one',
-    },
+    aliases: {},
```

Comment:
Do we have a way to check that these alias removals aren't breaking any downstream package consuming configs?

Reply:
I don't know